frontend/src/components/Navbar.css

```diff
@@ -0,0 +1,119 @@
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    background: transparent;
+    font-size: 1.25rem;
+}
+
+.navbar-logo {
+    display: flex;
+    align-items: center;
+    font-size: 2rem;
```

Comment:
font is too much bigger, keep it to 1.6rem

Reply:
Changed the `font-size` to 1.6rem.
![image](https://github.com/user-attachments/assets/b02e2cf9-0a13-40a6-9000-6f30ea7b7668)